src/Symfony/Bundle/FrameworkBundle/Validator/ConstraintValidatorFactory.php

```diff
@@ -11,7 +11,7 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Validator;
 
-use Symfony\Component\DependencyInjection\ContainerInterface;
+use Psr\Container\ContainerInterface;
```

Comment:
Not sure. ContainerInterface contains more methods than the Psr one, calling `$this->container->getParameter()` would break. We had to write a BC layer for a similar case in #21451